pkg/tfbridge/diff.go

```diff
@@ -111,10 +111,15 @@ func visitPropertyValue(name, path string, v resource.PropertyValue, tfs shim.Sc
 					}
 				}
 
-				ti = strconv.FormatInt(int64(tfs.SetHash(ev)), 10)
-				if containsComputedValues(e) {
-					// TF adds a '~' prefix to the hash code for any set element that contains computed values.
-					ti = "~" + ti
+				if !e.IsComputed() {
```

Comment:
nit: you could fold this into the body of the `if !e.IsComputed() && !e.IsOutput()` statement above